src/index.js

```diff
@@ -17,6 +17,7 @@ ReactDOM.render(
           path="/"
           exact
           element={(
+            // Import Redux Provider and the store in the main component of the app.
             <Provider store={store}>
               <Books />
 
```

Comment:
- [ ] As required for the project, kindly make sure the Main React component is wrapped in Provider and assigned `store`. I just noticed you didn't use the `App.js` as the main component, as is the case you should wrap all the code from `line 13 - 35`  in the `Provider`. All components are supposed to be handled with `Redux` and not only the `Books` component.